recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  string: h{{ PKG_HASH }}_py{{ python | replace(".", "") }}_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
+  number: 2
+  string: pyh{{PKG_HASH}}_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
```

Comment:
Here, spaces are needed between PKG_HASH and {{ and }}. Also, add tag `noarch: python` the way it is in other noarch packages.